docs/arcaflow/getting-started.md

```diff
@@ -1,101 +1,97 @@
 # Arcaflow Getting Started Guide
 
-## Step 1: Get a container engine
+## Running Workflows
+An Arcaflow workflow is a definition of steps structured together to perform complex actions. Workflows are defined as machine-readable YAML and therefore can be version-controlled and shared easily to run in different environments. A workflow is a way of encapsulating and sharing expertise and ensuring reproducible results.
 
-In order to use Arcaflow, you will need a container engine on your computer. For the purposes of this guide, we'll assume you are using Docker or Podman.
+Running a workflow only requires having the [Arcaflow engine binary](https://github.com/arcalot/arcaflow-engine/releases), the workflow definition file, and in most cases an input file. A config file is also optional and in some cases needed, depending on the target of the workflow. All of these files are typically defined in YAML. Additionally, the target of the workflow needs a compatible container platform, such as Podman, Docker, or Kubernetes.
```

Comment:
"is also optional and in some cases needed" is a pretty wishy-washy statement which I don't think actually tells the reader anything beyond "there's this thing called a config file".